### PR TITLE
Package prooftree.0.14

### PR DESCRIPTION
**Pull request by hendriktews**
### `prooftree.0.14`
Proof-tree visualization for Coq via Proof General
Prooftree visualizes proof trees during proof development in Coq with
Proof General. Prooftree helps against getting lost between different
subgoals in interactive proof development. It clearly shows where the
current subgoal comes from and thus helps in developing the right plan
for solving it.

---
* Homepage: http://askra.de/software/prooftree/
* Source repo: git+https://github.com/hendriktews/prooftree.git
* Bug tracker: https://github.com/hendriktews/prooftree/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

File: packages/prooftree/prooftree.0.14/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+subgoals in interactive proof development. It clearly shows where the
+current subgoal comes from and thus helps in developing the right plan
+for solving it.
+"""
+maintainer: "Hendrik Tews <hendrik@askra.de>"
+authors: "Hendrik Tews <hendrik@askra.de>"
+license: "GPL-3.0-or-later"
+homepage: "http://askra.de/software/prooftree/"
+bug-reports: "https://github.com/hendriktews/prooftree/issues"
+dev-repo: "git+https://github.com/hendriktews/prooftree.git"
```
Comment (haochenx) [inline packages/prooftree/prooftree.0.14/opam:15]: seems like a typo
```suggestion
dev-repo: "git+https://github.com/hendriktews/proof-tree.git"
```

File: packages/prooftree/prooftree.0.14/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Proof General. Prooftree helps against getting lost between different
+subgoals in interactive proof development. It clearly shows where the
+current subgoal comes from and thus helps in developing the right plan
+for solving it.
+"""
+maintainer: "Hendrik Tews <hendrik@askra.de>"
+authors: "Hendrik Tews <hendrik@askra.de>"
+license: "GPL-3.0-or-later"
+homepage: "http://askra.de/software/prooftree/"
+bug-reports: "https://github.com/hendriktews/prooftree/issues"
```
Comment (haochenx) [inline packages/prooftree/prooftree.0.14/opam:14]: seems like a typo
```suggestion
bug-reports: "https://github.com/hendriktews/proof-tree/issues"
```

File: packages/prooftree/prooftree.0.14/opam [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Prooftree visualizes proof trees during proof development in Coq with
+Proof General. Prooftree helps against getting lost between different
+subgoals in interactive proof development. It clearly shows where the
+current subgoal comes from and thus helps in developing the right plan
+for solving it.
+"""
+maintainer: "Hendrik Tews <hendrik@askra.de>"
+authors: "Hendrik Tews <hendrik@askra.de>"
+license: "GPL-3.0-or-later"
+homepage: "http://askra.de/software/prooftree/"
```
Comment (haochenx) [inline packages/prooftree/prooftree.0.14/opam:13]: It's entirely up to you, but HTTPS is probably better for the sake of security
```suggestion
homepage: "https://askra.de/software/prooftree/"
```

**Review by haochenx — CHANGES_REQUESTED** [review]
other than the inline comments, this PR looks good to me

**Comment by mseri** [comment]
Thanks. Please upstream the fixes to your opam file

**Comment by hendriktews** [comment]
Done, thanks.

**Pull request merged**